src/block_loss_functions.rs

```diff
@@ -114,16 +113,15 @@ impl BlockTrait for BlockSigmoid {
         debug_assert!(self.output_offset != usize::MAX);
 
         unsafe {
-            let wsum: f32 = {
-                let myslice = &pb
-                    .tape
-                    .get_unchecked(self.input_offset..(self.input_offset + self.num_inputs));
-                myslice.iter().sum()
-            };
+            let wsum: f32 = pb
+                .tape
+                .get_unchecked(self.input_offset..(self.input_offset + self.num_inputs))
+                .iter()
+                .sum();
             // vowpal compatibility
```

Comment:
Given this is cleanup PR, perhaps we can remove this comment too (there won't be any other way anytime soon)

Reply:
Removed.